rotkehlchen/chain/ethereum/constants.py

```diff
@@ -11,3 +13,7 @@
 ZERO_ADDRESS = string_to_ethereum_address('0x0000000000000000000000000000000000000000')
 
 CPT_KRAKEN = 'kraken'
+
+GENESIS = '0x0000000000000000000000000000000000000000'
```

Comment:
1. I was confused when I couldn't find `ZERO_ADDRESS` in the codebase during the previous review.  But here it is! So I think that you should replace `GENESIS_ADDR` with `ZERO_ADDRESS`